### PR TITLE
Add plot function for measurement chain

### LINKED ISSUE #65: add .plot() function to measurement chains
Add a simple plot/visualization function for measurement chains

Ideally based on NetworkX to prepare for #63

### PR DESCRIPTION
**Pull request by vhirtham**
## Changes

Adds a plot function for the measurement chain

## Related Issues

Closes #65

## Checks

- [x] updated CHANGELOG.md
- [x] ~~updated tests~~
- [x] updated doc/
- [x] ~~update example/tutorial notebooks~~ (postponed)

### CODE REVIEW

**Comment by vhirtham** [comment]
Silenced a Deepsource issue since the problem was introduced earlier, isn't probably too fast to fix, and also currently not that relevant.

File: weldx/measurement.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
 """Contains measurement related classes and functions."""
 
 from dataclasses import dataclass, field
-from typing import List, Union  # noqa: F401
+from typing import Dict, List, Tuple, Union  # noqa: F401
 
+import matplotlib.pyplot as plt
```
Comment (marscher) [inline weldx/measurement.py:6]: please import matplotlib and networkx lazily (inside the method bodies) as these pkgs dramatically increase the import time of weldx.

**Comment by marscher** [comment]
It would be great, if you could include a measurement plot somewhere in the notebooks ;)

**Comment by vhirtham** [comment]
> It would be great, if you could include a measurement plot somewhere in the notebooks ;)

It is planned, but we are currently rushing some features for the presentation next week. 😛

**Comment by vhirtham** [comment]
Will correct the documentation related issues tomorrow

**Review by CagtayFabry — APPROVED** [review]
LGTM, thanks !

merge after changelog entry and updating PR description

**Comment by CagtayFabry** [comment]
The flake8 issues go away if using flake8=3.8.x

File: weldx/measurement.py [hunk 2]
```diff
@@ -98,7 +97,7 @@ def _add_node(
         if parent_node is not None:
             graph.add_edge(parent_node, node)
 
-    def plot(self, axes: "Axes" = None):  # pragma: no cover
```
Comment (vhirtham) [inline weldx/measurement.py]: Why is it removed? We can simply fix the doc issue by adding it to the `nitpick_ignore`

Comment (CagtayFabry) [reply weldx/measurement.py]: it didn't work for me in sphinx but it works in other places (geometry.py)

I think it is pretty self-explanatory as is so we can fix that in a later PR if necessary

**Review by CagtayFabry — APPROVED** [review]

**Pull request merged**